tangelo/toolboxes/operators/tests/test_operators.py

```diff
@@ -73,6 +74,71 @@ def test_get_coeff(self):
         np.testing.assert_array_almost_equal(ref_one_body, one_body)
         np.testing.assert_array_almost_equal(ref_two_body, two_body)
 
+    def test_add_FermionOperator(self):
```

Comment:
1. The test class name already mention FermionOperator. You do not particularly need to repeat that into your test name, it is implicit.
2. Please add explicit tests for `__eq__` and `__neq__` in a new `test_eq` (using `==` and `!=`)